.ci/README.md

```diff
@@ -0,0 +1,179 @@
+
+See [our composite actions definitions](actions).
+
+Thanks to this composite actions strategy most of the Github Action jobs are basically the same and build-chain tool is in charge of making "the magic":
+
+* [droolsjbpm-build-boostrap](../.github/workflows/pull_request.yml)
+* [appformer](https://github.com/kiegroup/appformer/blob/main/.github/workflows/pull_request.yml)
+* [jbpm](https://github.com/kiegroup/jbpm/blob/main/.github/workflows/pull_request.yml)
+
+
+## Where can I find the commands which are going to be executed?
+
+All the build commands are listed in the [project-definition file](pull-request-config.yaml). [More information about project definition files](https://github.com/kiegroup/build-chain-configuration-reader). Just to summarize it, you will see something like:
+
+```
+  build-command:
+    current: whatever command for current.
+    upstream: whatever command for upstream.
+    downstream: whatever command for downstream.
+```
+
+* `current` means the command/s to be executed by the project triggering the job (or starting-project)
+* `upstream` means the command/s to be executed by the dependant projects of the project triggering the job (or starting-project).
+* `downstream` means the command/s to be executed by the projects that depend on the project triggering the job (or starting-project).
+
+You can find them on the default section (whether this section exists) or in the specific project section (whether this section exists, otherwise the information will be taken from the default section).
+
+More information about this at [github-action-build-chain#usage-example](https://github.com/kiegroup/github-action-build-chain#usage-example)
+
+> **_Note:_** You can always check the "Execution Summary" from any Jenkins or Github Actions job.
+
+## How are the projects related between them?
+
+As we have commented above, we delegate this part to build-chain tool and the project depdency information is externalize in one single file. See [project-dependencies.yaml file](https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/.ci/project-dependencies.yaml).
+
+There you will find a list of projects (all the projects involved on the different RHBA products, 7.x, 8.x ...) and the dependencies between them. For instance
+
+```
+  - project: kiegroup/lienzo-core
+
+  - project: kiegroup/lienzo-tests
+    dependencies:
+      - project: kiegroup/lienzo-core
+```
+
+means `kiegroup/lienzo-tests` project is part of the RHBA project set and depends on `kiegroup/lienzo-core` project (with no additional dependencies) which is also defined right on top of it.
+
+> **_Note:_** So in case `lienzo-tests` pull request is open a job checking out `lienzo-core` and `lienzo-tests` will be launched, building `lienzo-core` first and then `lienzo-tests`.
```

Comment:
```suggestion
> **_Note:_** So in case `lienzo-tests` pull request is opened the new PR job checks out `lienzo-core` and `lienzo-tests` PR CI job will be launched, where it is built `lienzo-core` repository project first and then `lienzo-tests` repository project.
```

Reply:
```suggestion
> **_Note:_** So in case `lienzo-tests` pull request is opened the new PR job checking out `lienzo-core` and `lienzo-tests` will be launched, where it is built `lienzo-core` repository project first and then `lienzo-tests` repository project.
```
wdyt @mareknovotny ?

Reply:
it does sound weird that part about `job checking out ...` I think my suggestion was to say that it is checked out or `that PR CI job checks out`